contracts/impls/MoneyMarket.sol

```diff
@@ -106,10 +111,10 @@ contract MoneyMarket is MoneyMarketInterface, Ownable, ReentrancyGuard {
     function rebalance() external nonReentrant {
         _rebalance(0);
     }
-
-    function setMinLiquidity(uint value) public onlyOwner {
-        require(value >= 0 && value <= Percentage.one(), "Invalid minLiquidity");
-        minLiquidity.value = value;
+	
+    function setMinLiquidity(uint256 _newMinLiquidity) public onlyOwner {
+        require(_newMinLiquidity >= 0 && _newMinLiquidity <= Percentage.one(), "Invalid minLiquidity");        
```

Comment:
can you install editorconfig plugin to trim extra spaces

Reply:
It's already in it, that's why the build failed.